docs/static/monitoring/monitoring-mb.asciidoc

```diff
@@ -8,12 +8,14 @@
 
 You can use {metricbeat} to collect data about {ls} and ship it to the
 monitoring cluster. The benefit of Metricbeat collection is that the monitoring
-agent remains active even if the {ls} instance does not. 
+agent remains active even if the {ls} instance does not.
+
+This step requires {ref}/configuring-metricbeat.html[{es} with {metricbeat} monitoring setup].
 
 To collect and ship monitoring data:
 
 . <<disable-default,Disable default collection of monitoring metrics>>
-. <<define-cluster__uuid,Specify the target `cluster_uuid` (optional)>>
+. <<define-cluster__uuid,Specify the target `cluster_uuid`>>
```

Comment:
review note: intentionally removing _(optional)_ that LS metrics will not be displayed under the cluster with metricbeat monitoring.